documentation/markdown/architecture/features/protocol/authorization.md

```diff
@@ -35,21 +36,21 @@ The diagram below shows the default configuration if authentication is enabled.
 flowchart TD
   CredentialsExtractor("<strong>CredentialsExtractor</strong><br>UnionCredentialsExtractor")
   CredentialsExtractor --> CredentialsExtractorArgs
-  
+
   subgraph CredentialsExtractorArgs[" "]
     WaterfallHandler("<br>WaterfallHandler")
     PublicCredentialsExtractor("<br>PublicCredentialsExtractor")
   end
-  
+
   WaterfallHandler --> WaterfallHandlerArgs
   subgraph WaterfallHandlerArgs[" "]
     direction LR
     DPoPWebIdExtractor("<br>DPoPWebIdExtractor") --> BearerWebIdExtractor("<br>BearerWebIdExtractor")
   end
 ```
 
-Both of the WebID extractors make use of 
-the (`access-token-verifier`)[https://github.com/CommunitySolidServer/access-token-verifier] library
+Both of the WebID extractors make use of
+the [`access-token-verifier`](https://github.com/CommunitySolidServer/access-token-verifier) library
```

Comment:
Guess this shows how many people actually read the documentation that nobody commented on this yet 😄